packages/@n8n/task-runner-python/src/constants.py

```diff
@@ -19,7 +19,7 @@
 DEFAULT_MAX_CONCURRENCY = 5  # tasks
 DEFAULT_MAX_PAYLOAD_SIZE = 1024 * 1024 * 1024  # 1 GiB
 DEFAULT_TASK_TIMEOUT = 60  # seconds
-DEFAULT_AUTO_SHUTDOWN_TIMEOUT = 15  # seconds
+DEFAULT_AUTO_SHUTDOWN_TIMEOUT = 0  # seconds
```

Comment:
The launcher [defaults the idle timeout to 15s](https://github.com/n8n-io/task-runner-launcher/blob/4b175283884cb6471508f91f0fd1e2f4113eb9ee/internal/config/config.go#L40) and runner config [passes it along](https://github.com/n8n-io/n8n/blob/efc3a2d66484a14338cb67aa1135e71d25fb5c4e/docker/images/runners/n8n-task-runners.json#L40) but the [default idle timeout in JS is disabled](https://github.com/n8n-io/n8n/blob/efc3a2d66484a14338cb67aa1135e71d25fb5c4e/packages/%40n8n/task-runner/src/config/base-runner-config.ts#L40), so let's follow suit.